### PR TITLE
Spring Boot web task is completed

### PR DESCRIPTION
**Pull request by azarovartur**

### CODE REVIEW

File: src/main/java/mate/academy/bookstore/BookMapper.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package mate.academy.bookstore;
```
Comment (sokimaaa) [inline src/main/java/mate/academy/bookstore/BookMapper.java]: let's move all Mapper classes into the specific package, e.g. `mapper`.

File: src/main/java/mate/academy/bookstore/dto/BookDto.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package mate.academy.bookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```
Comment (sokimaaa) [inline src/main/java/mate/academy/bookstore/dto/BookDto.java]: let's use just @Data annotation for DTO's

**Review by sokimaaa — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/bookstore/BookMapper.java (hunk 1 above)
Comment (azarovartur) [reply src/main/java/mate/academy/bookstore/BookMapper.java]: done

File: src/main/java/mate/academy/bookstore/dto/BookDto.java (hunk 2 above)
Comment (azarovartur) [reply src/main/java/mate/academy/bookstore/dto/BookDto.java]: done

**Review by aaananas — APPROVED** [review]

**Pull request merged**